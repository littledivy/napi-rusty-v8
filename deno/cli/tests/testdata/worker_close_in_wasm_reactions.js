// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.

// https://github.com/denoland/deno/issues/12263
// Test for a panic that happens when a worker is closed in the reactions of a
// WASM async operation.

new Worker(
  new URL("./workers/close_in_wasm_reactions.js", import.meta.url),
  { type: "module" },
);
