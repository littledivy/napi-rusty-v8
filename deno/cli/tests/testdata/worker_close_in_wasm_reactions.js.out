Error: CompileError: WebAssembly.compile(): expected string length @+10
